ios/Exponent/Kernel/AppLoader/CachedResource/EXManifestResource.h

```diff
@@ -28,7 +28,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (NSError *)verifyManifestSdkVersion:(EXManifestsManifest *)maybeManifest;
 - (NSError *)formatError:(NSError *)error;
-
++ (NSString *)formatHeader:(NSError *)error;
++ (NSAttributedString *)addErrorStringHyperlinks:(NSString *) errorString;
```

Comment:
```suggestion
+ (NSAttributedString *)addErrorStringHyperlinks:(NSString *)errorString;
```

Reply:
Could you please use `nullable`/`nonnull` annotations for these types?